README.md

```diff
@@ -30,6 +30,17 @@ and file location that are the cause of the problem, as well as a link
 to the implementation of that query in the current version of the tool.
 
 ## FAQ
+- [What Rust versions does `cargo-semver-checks` support?](#what-rust-versions-does-cargo-semver-checks-support)
+- [Can I use `cargo-semver-checks` with `nightly` Rust?](#can-i-use-cargo-semver-checks-with-nightly-rust)
+- [What if my project needs stronger guarantees around supported Rust versions?](#what-if-my-project-needs-stronger-guarantees-around-supported-rust-versions)
+- [Does the crate I'm checking have to be published on crates.io?](#does-the-crate-im-checking-have-to-be-published-on-cratesio)
+- [What features does `cargo-semver-checks` enable in the tested crates?](#what-features-does-cargo-semver-checks-enable-in-the-tested-crates)
+- [Does `cargo-semver-checks` have false positives?](#does-cargo-semver-checks-have-false-positives)
+- [Will `cargo-semver-checks` catch every semver violation?](#will-cargo-semver-checks-catch-every-semver-violation)
+- [If I really want a new feature to be implemented, can I sponsor its development?](#if-i-really-want-a-new-feature-to-be-implemented-can-i-sponsor-its-development)
+- [How is `cargo-semver-checks` similar to and different from other tools?](#how-is-cargo-semver-checks-similar-to-and-different-from-other-tools)
+- [Why is it sometimes `cargo-semver-check` and `cargo-semver-checks`?](#why-is-it-sometimes-cargo-semver-check-and-cargo-semver-checks)
```

Comment:
Outdated? Now it's consistently `-checks` everywhere.

Reply:
It's consistent _almost_ everywhere: https://crates.io/crates/cargo-semver-check

I was on the fence about removing it but decided to keep it for a bit longer. I'll probably remove it the next time I touch up the README.